packages/components/select/__tests__/select.test.ts

```diff
@@ -161,7 +161,11 @@ const getSelectVm = (configs: SelectProps = {}, options?) => {
   )
 }
 
-const getGroupSelectVm = (configs: SelectProps = {}, options?) => {
+const getGroupSelectVm = (
+  configs: SelectProps = {},
+  options?,
+  groupTemplate?
+) => {
```

Comment:
This seems redundant, you should keep the data format consistent with options, and modify the options through `wrapper.vm.options`.